spec/src/main/asciidoc/WebSocket.adoc

```diff
@@ -245,7 +245,7 @@ safely use this method is when you are directly implementing
 class. This approach guarantees that generic type information will be
 present in the generated class file and the runtime will be able to get
 it. For any other case (Lambda Expressions included), one of following
-methods have to be used:
+methods has to be used:
```

Comment:
meh, kinda liked the old one better.
doesn't really improve readability.

how about "one of the methods must be used:" ?